docs/csharp/fundamentals/program-structure/main-command-line.md

```diff
@@ -29,47 +29,50 @@ For more information, see [Top-level statements](top-level-statements.md).
 ## Overview
 
 - The `Main` method is the entry point of an executable program; it is where the program control starts and ends.
-- `Main` is declared inside a class or struct. `Main` must be [`static`](../../language-reference/keywords/static.md) and it need not be [`public`](../../language-reference/keywords/public.md). (In the earlier example, it receives the default access of [`private`](../../language-reference/keywords/private.md).) An enclosing `class` can be `static`.
+- `Main` must be declared inside a class or struct. The enclosing `class` can be `static`.
+- `Main` must be [`static`](../../language-reference/keywords/static.md).
+- `Main` can have a [`public`](../../language-reference/keywords/public.md), [`protected`](../../language-reference/keywords/protected.md), [`private`](../../language-reference/keywords/private.md), [`protected internal`](../../language-reference/keywords/protected-internal.md) and [`private protected`](../../language-reference/keywords/private-protected.md) access modifier. If not specified, it receives the default access of `private`.
```

Comment:
This feels unnecessary. If you accept my edit on line 52, this bullet can be deleted.

Reply:
I replaced negations with affirmations in this list, because I like to see explicit and clear rules.
Can you consider to keep this list?

Reply:
I prefer to keep this bullet about access modifier, because it details the contract.

Reply:
I am going to push back again on this one. The rule is that *any* access modifier is legal, and if none is specified, the default is `private`. That's the same rule as any member. 

The way you've written it implies `Main` is different. And, it can lead to subtle mistakes. (There's one already in that `Main` can have `internal` access).  See [Sharplab](https://sharplab.io/#v2:EYLgtghglgdgNAFxAJwK4wD4AEBMAGAWAChcBGY43AZmIG9iACJh2BAU2RggBsGtSAbHwAsDALLQYACgCUjZvSLNlfUgE4pAIkA8G4BB9zTIDc8pgF9i5okA).

Reply:
I forget `internal`, nice catch.
I rewrite this part, so it's less heavy. @BillWagner,  what do you think?

Reply:
I like the update. Approved, and I'll merge now.